internal/backends/winit/renderer/sw.rs

```diff
@@ -45,25 +45,12 @@ impl super::WinitCompatibleRenderer for WinitSoftwareRenderer {
     }
 
     fn render(&self, window: &i_slint_core::api::Window) -> Result<(), PlatformError> {
-        let size = window.size();
-
-        let width = size.width.try_into().map_err(|_| {
-            format!(
-                "Attempting to resize softbuffer window surface with an invalid width: {}",
-                size.width
-            )
-        })?;
-        let height = size.height.try_into().map_err(|_| {
-            format!(
-                "Attempting to resize softbuffer window surface with an invalid height: {}",
-                size.height
-            )
-        })?;
+        let size: NonZeroPhysicalSize = window.size().try_into()?;
```

Comment:
IMHO an empty window should just render nothing and not be an error that will result in a panic later.

Reply:
Agreed. Will change accordingly.